docs/source/developer/interactives.rst

```diff
@@ -49,29 +49,42 @@ Interactive Configuration Files
 
 - **Required Fields:**
 
-  - ``interactives:`` A dictionary of all interactive data, where each key is a slug for an interacive.
+  - ``interactives:`` A dictionary of all interactive data, where each key is a slug for an interacive. 
+TODO: Check below wording
+  - The dictionary of interactive data must include:
+
+    - ``languages:`` A dictionary of all languages the interactive is available in, where each key is the locale.
+
+    - ``is_interactive:`` A boolean value indicating if the interactive is interactable. This field was introduced
```

Comment:
`interactable` is not a word. Unfortunately the word you should use is `interactive`, or alternatively `can be interacted with`

Reply:
Also perhaps mention the term `uninteractive` that we use for an non-interactive interactive, referring to further down in the process

Reply:
Lol look at me making up words. Thanks!